### PR TITLE
Cknieling/maya2022

### PR DESCRIPTION
**Pull request by chr11115**
- Updated makefiles and CI to support maya 2022
- related internal PR that needs to be merged, before we can switch the pipeline back to development/production

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ import groovy.transform.Field
 
 // -- PIPELINE LIBRARIES
 
-@Library('psl')
+@Library('psl@chr11115/maya2022')
```
Comment (mistafunk) [inline Jenkinsfile:8]: remember to revert this before merge

File: Jenkinsfile [hunk 2]
```diff
@@ -14,7 +14,7 @@ import com.esri.zrh.jenkins.PslFactory
 
 // -- SETUP
 
-psl.runsHere('production')
+psl.runsHere('testing')
```
Comment (mistafunk) [inline Jenkinsfile:17]: same, remember to revert

**Review by mistafunk — APPROVED** [review]
LGTM. please test it once more also in the 'development' CI environment before merging.

**Comment by chr11115** [comment]
Tested on development, after passing all tests, moved to production/main.

**Pull request merged**